internal/controllers/language_detect/language_detect.go

```diff
@@ -135,12 +133,12 @@ func (ld *LanguageDetect) filesAndFoldersToIgnore(path string) bool {
 
 func (ld *LanguageDetect) checkDefaultPathsToIgnore(path string) bool {
 	for _, value := range toignore.GetDefaultFoldersToIgnore() {
-		if strings.Contains(path, file.ReplacePathSeparator(value)) {
+		if strings.Contains(path, value) {
 			return true
 		}
 	}
 	if !ld.configs.EnableGitHistoryAnalysis {
```

Comment:
I've reviewed the validations and it's really right!
But we have to be careful, as GitLeaks only starts its analysis if it has the `.git` folder inside `.horusec`
It may happen that the horusec-leaks tool finds a vulnerability because there are many "hashes" in this folder `.git` that generate false positives